package.json

```diff
@@ -27,10 +27,13 @@
     "prebuild": "npm run doc-listing-inject",
     "build": "node scripts/build.js",
     "postbuild": "rm -rf examples/node_modules && mkdir examples/node_modules && cp -r node_modules/@webcomponents node_modules/requirejs examples/node_modules/",
-    "clean": "rm -rf node_modules package-lock.json elements/*/{node_modules,package-lock.json,_temp} test/vrt-snapshots",
+    "clean": "rm -rf node_modules elements/*/{node_modules,package-lock.json}",
```

Comment:
It also looks like you're no longer removing the root `package-lock.json`. Is this intentional?

Reply:
Yeah, the package-lock is committed and is our source of truth for shared versioning for packages - I don't think we should delete it on install